apps/confidential/confidential.go

```diff
@@ -92,19 +91,28 @@ func CertFromPEM(pemData []byte, password string) ([]*x509.Certificate, crypto.P
 		case "CERTIFICATE":
 			cert, err := x509.ParseCertificate(block.Bytes)
 			if err != nil {
-				return nil, nil, fmt.Errorf("block labelled 'CERTIFICATE' could not be pared by x509: %w", err)
+				return nil, nil, fmt.Errorf("block labelled 'CERTIFICATE' could not be parsed by x509: %w", err)
 			}
 			certs = append(certs, cert)
 		case "PRIVATE KEY":
 			if priv != nil {
-				return nil, nil, fmt.Errorf("found multiple blocks labelled 'PRIVATE KEY'")
+				return nil, nil, errors.New("found multiple private key blocks")
 			}
 
 			var err error
 			priv, err = parsePrivateKey(block.Bytes)
```

Comment:
nit: might be good to rename 'parsePrivateKey' now that we are supporting more than one private key type

Reply:
I inlined it.